x-pack/plugins/endpoint/public/sagas/home.ts

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { AppMountContext } from 'kibana/public';
+import { withPageNavigationStatus } from './common';
+import { hrefIsForPath } from '../concerns/routing';
+import { actions as homeActions } from '../actions/home';
+
+// TODO: type this properly
+export async function homeSaga(...args: any[]) {
+  await Promise.all([resourceSaga(...args)]);
+}
+
+// TODO type actionsAndState, dispatch
+async function resourceSaga(
+  { actionsAndState, dispatch }: { actionsAndState: any; dispatch: any },
```

Comment:
With the latest code (upstream) you should now have a type in `../lib/saga` named `SagaContext`. use that here:

```typescript
{ actionAndState, dispatch }: SagaContext
```

Reply:
Ah ok thanks, updated it 👍